source/mongodump.txt

```diff
@@ -60,6 +60,65 @@ Run ``mongodump`` from the system command line, not the
 
 .. include:: /includes/fact-online-archive-callout.rst
 
+``mongodump`` can export:
+
+- Collection documents, metadata, and options.
+- Index definitions.
+- Writes that occur during the export.
```

Comment:
Looks good! My only suggestion is to drop "can" so it's clear up front what's exported and what's optional

```suggestion
``mongodump`` exports:

- Collection documents, metadata, and options.
- Index definitions.
- Writes that occur during the export, if run with the ``mongodump`` :option:`--oplog <mongodump --oplog>` option.
```

Reply:
Done.